### PR TITLE
Search restaurants by name or a sub string inside the name of restaurants

### PR DESCRIPTION
**Pull request by vincenzopalazzo**

### CODE REVIEW

**Review by francescofact — APPROVED** [review]
@francescofact reviewed. keep up the good work!

File: monolith/database.py [hunk 1]
```diff
@@ -212,6 +212,8 @@ class MenuPhotoGallery(db.Model):
     menu = relationship("Menu", foreign_keys="MenuPhotoGallery.menu_id")
 
 
+## Pysql doesnt't support the floating point and we need to support it
```
Comment (vincenzopalazzo) [inline monolith/database.py:215]: I need to move it in a read doc comment in python

File: monolith/services/restaurant_services.py [hunk 2]
```diff
@@ -190,6 +190,9 @@ def get_restaurant_people(restaurant_id: int):
             )
             .first()
         )
+        if openings is None:
```
Comment (vincenzopalazzo) [inline monolith/services/restaurant_services.py:193]: good catch :-)

**Review by vincenzopalazzo — COMMENTED** [review]
Thanks to fix the bug inside the test and inside my search method

**Pull request merged**